portal-api/src/v1/controllers/reports/review-ukef-decision.controller.js

```diff
@@ -67,7 +67,7 @@ const getUkefDecision = async (decision, bankId) => {
         },
       },
     },
-    { $sort: { dateOfApprovalEpoch: 1 } }
+    { $sort: { dateOfApprovalEpoch: 1 } } // TODO SR-8: Sort order -- no $eq expression needed
```

Comment:
Remove todo once noted:
```suggestion
    { $sort: { dateOfApprovalEpoch: 1 } }
```